lms/djangoapps/teams/static/teams/js/views/team_card.js

```diff
@@ -19,22 +19,36 @@
                 '<span class="member-count"><%= membership_message %></span>' +
                 '<ul class="list-member-thumbs"></ul>'
             ),
+            profileImageTemplate: _.template(
```

Comment:
I agree. Currently we have a generic card view which owns a generic card template. Our team card provides a list of detail views which spit out some markup which is added to the generic template by the generic card. Without a major overhaul of our generic card I think the option that makes the most sense is to separate each occurrence of `_.template(<markup>)` into their own underscore files.